master/internal/user/postgres_users.go

```diff
@@ -55,6 +55,13 @@ func UserByUsername(username string) (*model.User, error) {
 	return &user, nil
 }
 
+// Set DisplayName in User.
+func SetDisplayName(userID int32, displayName string) error {
+	var user model.User
+	_, err := db.Bun().NewUpdate().Model(&user).Set("display_name = ?", displayName).Where("id = ?", userID).Exec(context.TODO())
```

Comment:
`.Model((*model.User)(nil)).` is a way to do the same without a separate `var` declaration